spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationExecution.scala

```diff
@@ -0,0 +1,256 @@
+    val columnNameOfCorruptRecord = SparkUtils.setUniqueColumnNameOfCorruptRecord(spark, schema)
+    if (cmd.rawFormat.equalsIgnoreCase("fixed-width") || cmd.failOnInputNotPerSchema) {
+      None
+    } else {
+      Option(columnNameOfCorruptRecord)
+    }
+  }
+
+  protected def standardize(inputData: DataFrame, schema: StructType, cmd: StandardizationConfig)
+                           (implicit spark: SparkSession, udfLib: UDFLibrary): DataFrame = {
+    //scalastyle:on parameter.number
+    val recordIdGenerationStrategy = getRecordIdGenerationStrategyFromConfig(conf)
+
+    try {
+      handleControlInfoValidation()
+      StandardizationInterpreter.standardize(inputData, schema, cmd.rawFormat,
+        cmd.failOnInputNotPerSchema, recordIdGenerationStrategy)
+    } catch {
+      case e@ValidationException(msg, errors) =>
+        val errorDescription = s"$msg\nDetails: ${errors.mkString("\n")}"
+        AtumImplicits.SparkSessionWrapper(spark)
+          .setControlMeasurementError("Schema Validation", errorDescription, "")
+        throw e
+      case NonFatal(e) if !e.isInstanceOf[ValidationException] =>
+        val sw = new StringWriter
+        e.printStackTrace(new PrintWriter(sw))
+        AtumImplicits.SparkSessionWrapper(spark).setControlMeasurementError(sourceId.toString, e.getMessage, sw.toString)
+        throw e
+    }
+  }
+
+  protected def processStandardizationResult(args: Array[String],
+                                             standardizedDF: DataFrame,
+                                             preparationResult: PreparationResult,
+                                             schema: StructType,
+                                             cmd: StandardizationConfig,
+                                             menasCredentials: MenasCredentials)
+                                            (implicit spark: SparkSession,
+                                             fsUtils: FileSystemVersionUtils): Unit = {
+    import za.co.absa.atum.AtumImplicits._
+    val fieldRenames = SchemaUtils.getRenamesInSchema(schema)
+    fieldRenames.foreach {
+      case (destinationName, sourceName) => standardizedDF.registerColumnRename(sourceName, destinationName)
+    }
+
+    standardizedDF.setCheckpoint(s"$sourceId - End", persistInDatabase = false)
+
+    val recordCount = standardizedDF.lastCheckpointRowCount match {
+      case None => standardizedDF.count
+      case Some(p) => p
+    }
+    if (recordCount == 0) {
+      handleEmptyOutput(sourceId)
+    }
+
+    standardizedDF.write.parquet(preparationResult.pathCfg.outputPath)
+    // Store performance metrics
+    // (record count, directory sizes, elapsed time, etc. to _INFO file metadata and performance file)
+    val stdDirSize = fsUtils.getDirectorySize(preparationResult.pathCfg.outputPath)
+    preparationResult.performance.finishMeasurement(stdDirSize, recordCount)
+    PerformanceMetricTools.addPerformanceMetricsToAtumMetadata(
+      spark,
+      "std",
```

Comment:
```suggestion
      "standardization",
```

Reply:
It makes sense. My only concern is if it will be inconsistent with the prefixes used in migration and monitoring or they should also be changed there

Reply:
Good catch. We might ignore it for now.

Reply:
These are only prefixes fo metadata so, let's do another PR if needed.